modules/core/src/lib/utils/mime-type-utils.js

```diff
@@ -29,5 +29,5 @@ export function parseMIMETypeFromURL(dataUrl) {
     return matches[1];
   }
 
-  return '';
+  return dataUrl;
```

Comment:
This doesn't make sense. `parseMimeType`'s input is already a valid mime type string. `dataUrl` is not.